C++/source/graph/Dijkstra.cpp

```diff
@@ -0,0 +1,49 @@
+// Author : Abdelrahman Hammam
+//Github handle : AbdoHammam
+#include<bits/stdc++.h>
```

Comment:
Don't include `<bits/stdc++.h>`. Only include those header files that your code will need.